src/Chapter1.hs

```diff
@@ -553,8 +542,11 @@ value after "=" where the condition is true.
 
 Casual reminder about adding top-level type signatures for all functions :)
 -}
+mid :: Int -> Int -> Int -> Int
+mid x y z
+    | x > y = if y > z then y else if x > z then z else x
+    | otherwise = if x > z then x else if y > z then z else y
```

Comment:
That's a nice solution!
I guess, you can use only guards here instead of mixing `guards` and `if-then-else` expressions, but that's just a stylistic preference :slightly_smiling_face: